mitodl/mit-open
# add the footer & privacy, terms and about us pages

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4247

### Description (What does it do?)
This PR adds a footer to the site based on the most recent designs, linked in the above issue. Some other static pages were also added that are linked to from the footer; About Us, Terms and Services and Privacy Policy.

### Screenshots (if appropriate):
<img width="959" alt="image" src="https://github.com/mitodl/mit-open/assets/12089658/6e445ffb-7984-49ea-b4c1-dd847d081ea7">
<img width="187" alt="image" src="https://github.com/mitodl/mit-open/assets/12089658/a799a8a2-572f-433f-831f-a7669365ca00">
<img width="190" alt="image" src="https://github.com/mitodl/mit-open/assets/12089658/80cb9f86-80c5-4e8f-86ee-c7316334f827">

### How can this be tested?
 - Spin up `mit-open` on this branch
 - Load the home page
 - Ensure that the footer appears consistent with the designs, linked in the issue above
 - Click the links and ensure that they go to the proper place, as defined in the issue above
 - Ensure that the footer and the pages linked therein are legible on mobile

### Additional Context
The About Us page has a desktop design as of the time of writing:

https://www.figma.com/proto/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?page-id=107%3A2479&node-id=3000-26336&viewport=5455%2C85%2C1.09&t=qq2zFcco3CVGwgBi-1&scaling=scale-down&starting-point-node-id=2197%3A31345&show-proto-sidebar=1

There is yet to be a mobile design for this page, or any for the privacy policy and terms pages. The latter two were mocked up from a Google doc. The About page should look like the design, but the others are just placeholders to get something in there.

I'm going to leave https://github.com/mitodl/hq/issues/4380 open for when we get the final designs for everything mentioned above, and we can work on it then.
